Week2/homework/maartjes_work.js

```diff
@@ -43,5 +43,70 @@ const tuesday = [
 ];
 
 const tasks = monday.concat(tuesday);
+/*
+function duration(item) {
+  return item.duration;
+}
+function sum(prev, next) {
+  return prev + next;
+}
+console.log(tasks.map(duration).reduce(sum));
+*/
+
+/*
+let hours = 0;
+for (let i = 0; i < tasks.length; i++) {
+  const num = tasks[i].duration;
+  hours += num;
+}
+hours /= 60;
+console.log(hours);
+*/
+
+let hours = 0;
+const duration = (tasks.map(tasks => tasks.duration));
+for (let i = 0; i < duration.length; i++) {
+  hours += duration[i];
+}
+hours /= 60;
+console.log(hours);
+
+//------------------
+
+
+/*
+const twoOrMoreHours = [];
+for (let i = 0; i < duration.length; i++) {
+  if (duration[i] >= 120) twoOrMoreHours.push(duration[i]);
+}
+console.log(twoOrMoreHours);
+*/
+
+
+const twoOrMoreHours = (duration.filter(values => {
+  return values >= 120;
+}
+));
+
+console.log(twoOrMoreHours);
+
+//------------------
+/*
+const salaryPerHour = '24.35';
+let salary = 0;
+
+for (let i = 0; i < duration.length; i++) {
+  salary = hours * salaryPerHour;
+}
+console.log('€ ' + salary.toFixed(2));
+
+*/
+
+const HoursArray = [hours];
+function add(sum, num) {
+  return sum + num;
+}
+const salary = HoursArray.map(HoursArray => HoursArray * 24.35).reduce(add, 0);
```

Comment:
Variable names should be in camelCase. You should apply the `.map()` method to the `twoOrMoreHours` array.

The number 24.35 comes out of the blue without explanation. Developers call this a `magic` number.  I assume it represents the hourly rate that Maartje earns. Therefore best to name it as such:

```js
const hourlyRate = 24.35;
```

```js
const salary = twoOrMoreHours
  .map(duration => duration * hourlyRate)
  .reduce(add, 0);
```